airflow/contrib/operators/kubernetes_operator.py

```diff
@@ -93,6 +97,8 @@ def __init__(self,
         self.cloudsql_instance_creds = 'airflow-cloudsql-instance-credentials'
         self.cloudsql_db_creds = 'airflow-cloudsql-db-credentials'
 
+        self.volumes = volumes or []
```

Comment:
Maybe the note on line 91 should go away?

Reply:
fair point